app/javascript/components/physical-storage-form/physical-storage-form.schema.js

```diff
@@ -59,13 +62,56 @@ const createSchema = (edit, ems, initialValues, state, setState) => {
             isDisabled: edit,
             validate: [{ type: validatorTypes.REQUIRED }],
             loadOptions: () => (emsId ? loadFamilies(emsId) : Promise.resolve([])),
+            onChange: (value) => setFamilyId(value),
             includeEmpty: true,
             key: `physical_storage_family_id-${emsId}`,
             condition: {
               when: 'ems_id',
               isNotEmpty: true,
             },
           },
+          {
+            component: componentTypes.RADIO,
+            id: 'capabilities',
+            name: 'capabilities',
+            label: __('Capabilities'),
+            isRequired: true,
+            validate: [{ type: validatorTypes.REQUIRED }],
+            options: [
+              {
+                label: 'Default',
+                value: 'Default',
+              },
+              {
+                label: 'Custom',
+                value: 'Custom',
+              },
```

Comment:
Labels need to be externalized `__('Default')`

Reply:
done